android-components/components/feature/search/src/main/java/mozilla/components/feature/search/telemetry/Utils.kt

```diff
@@ -113,28 +98,17 @@ private fun getTrackKeyFromCookies(
                 .split("=")
                 .map { item -> item.trim() }
 
-            if (valueList.size == 2 && valueList[0] == followOnCookie.codeParam &&
-                followOnCookie.codePrefixes.any { prefix ->
-                    valueList[1].startsWith(
-                        prefix,
-                    )
+            if (valueList.size == 2 && valueList[0] == followOnCookie.codeParamName &&
+                provider.taggedCodes.any { prefix ->
+                    valueList[1] == prefix
```

Comment:
I see this is `==` instead of `startsWith` now. Just double-checking that is intentional?